config/localization/en/app.json

```diff
@@ -676,7 +683,8 @@
     "LEGAL": {
       "ACADEMY_LINK_DESCRIPTION": "To access the dYdX Academy, you will leave the dYdX Trading Inc. website and join a website made available by dYdX Foundation, which is independent from and unaffiliated with dYdX Trading Inc. dYdX Trading Inc. is not responsible for any action taken or content on the third-party website.",
       "LEGAL": "Legal",
-      "GOVERNANCE_LINK_DESCRIPTION": "To participate in governance, you will leave the dYdX Trading Inc. website and join a website made available by dYdX Foundation, which is independent from and unaffiliated with dYdX Trading Inc. dYdX Trading Inc. is not responsible for any action taken or content on the third-party website.",
+      "GOVERNANCE_LINK_DESCRIPTION": "To participate in governance, you need to export your secret phrase to Keplr Wallet.",
```

Comment:
ah it looks like now it's a duplicate of the one inside TRADING_REWARDS? should we just keep one?

also where is this supposed to show up? i think these should be the most updated according to feedback by antonio:
<img width="1232" alt="image" src="https://github.com/dydxprotocol/v4-localization/assets/9400120/c245ae63-6bce-4a51-84e4-a6d3d3e089fa">